recipe/meta.yaml

```
@@ -27,55 +30,60 @@ requirements:
     - future
     - menuinst  # [win]
   run:
-    - python
+    - python >=3.6
```

This way of constraining the Python version should only be used for `noarch: python` packages. The version constraint here comes from using `skip: true  # [py27]`
```suggestion
    - python
```

---

ok